tests/report/reportportal/test.sh

```diff
@@ -14,6 +14,7 @@ rlJournalStart
     rlPhaseEnd
 
     rlPhaseStartTest
+	rlRun "export REQUESTS_CA_BUNDLE=/etc/pki/tls/certs/ca-bundle.crt"
```

Comment:
Hm, I'm not sure we should include this setup in the test. Isn't this rather a user config?

Reply:
Thanks.